pkg/ingester/flush.go

```diff
@@ -67,6 +67,10 @@ var (
 		Name: "cortex_ingester_dropped_chunks_total",
 		Help: "Total number of chunks dropped from flushing because they have too few samples.",
 	})
+	oldestUnflushedChunkTimestamp = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
I don't know what happened, but looks something went wrong with the last merge/rebase you did.